eclair-core/src/main/scala/fr/acinq/eclair/io/Peer.scala

```diff
@@ -72,20 +74,20 @@ class Peer(nodeParams: NodeParams, remoteNodeId: PublicKey, authenticator: Actor
           stay using d.copy(attempts = attempts + 1)
       }
 
-    case Event(Authenticator.Authenticated(_, transport, remoteNodeId, address, outgoing, origin_opt), DisconnectedData(_, channels, _)) =>
-      log.debug(s"got authenticated connection to $remoteNodeId@${address.getHostString}:${address.getPort}")
+    case Event(Authenticator.Authenticated(_, transport, remoteNodeId1, address, outgoing, origin_opt), DisconnectedData(_, channels, _)) =>
```

Comment:
I think what we really want to do here is make sure that the `Authenticated` object is indeed for the correct `remotenodeId`.

How adding:
 ```scala
require(remoteNodeId == remoteNodeId1, s"invalid nodeid: $remoteNodeId != remoteNodeId1")
```
and then keep using `remoteNodeId`.